### PR TITLE
[MODULAR] CME event balance/alert

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

- Now that we have engineering override, updates CME extreme+ to use orange alert, not red.
- Extreme CME is too powerful to run in normal games, so its power/range have been reduced, timer inceased.
- Other CME levels have lowered range and tighter timers. Generally more, smaller EMPs vs less larger ones.

## How This Contributes To The Skyrat Roleplay Experience

Orange is actually useful to engineers vs red. We can put CME extreme as a low frequency random event since it's not making an EMP the size of your screen.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl: LT3
balance: CME extreme and higher triggers orange alert, not red
balance: All CME range reduced, frequency increased
balance: CME extreme power reduced, explosion countdown timer increased
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/airlock_override/code/airlock_override.dm [hunk 1]
```diff
@@ -149,6 +149,27 @@ GLOBAL_VAR_INIT(force_eng_override, FALSE)
 		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_FORCE_ENG_OVERRIDE, FALSE)
 		return
 
+/proc/eng_override_on(auto = FALSE)
+	GLOB.force_eng_override = TRUE
+	minor_announce("Engineering staff will have expanded access to areas of the station during the emergency.", "Engineering Emergency")
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_FORCE_ENG_OVERRIDE, TRUE)
+	if(!auto)
+		SSblackbox.record_feedback("nested tally", "keycard_auths", 1, list("engineer override access", "enabled"))
+	return
```
Comment (Zonespace27) [inline modular_skyrat/modules/airlock_override/code/airlock_override.dm]: does nothing

File: modular_skyrat/modules/airlock_override/code/airlock_override.dm [hunk 2]
```diff
@@ -149,6 +149,27 @@ GLOBAL_VAR_INIT(force_eng_override, FALSE)
...
+/proc/eng_override_off(auto = FALSE)
+	GLOB.force_eng_override = FALSE
+	minor_announce("Expanded engineering access has been revoked.", "Engineering Emergency")
+	var/level = SSsecurity_level.get_current_level_as_number()
+	if(!auto)
+		SSblackbox.record_feedback("nested tally", "keycard_auths", 1, list("engineer override access", "disabled"))
+	if(level == SEC_LEVEL_ORANGE)
+		return
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_FORCE_ENG_OVERRIDE, FALSE)
+	return
```
Comment (Zonespace27) [inline modular_skyrat/modules/airlock_override/code/airlock_override.dm]: does nothing

File: modular_skyrat/modules/airlock_override/code/airlock_override.dm [hunk 3]
```diff
@@ -149,6 +149,27 @@ GLOBAL_VAR_INIT(force_eng_override, FALSE)
 		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_FORCE_ENG_OVERRIDE, FALSE)
 		return
 
+/proc/eng_override_on(auto = FALSE)
```
Comment (Zonespace27) [inline modular_skyrat/modules/airlock_override/code/airlock_override.dm]: having an `on`, `off`, and `toggle` proc feels unnecessary

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/airlock_override/code/airlock_override.dm (hunk 3 above)
Comment (lessthnthree) [reply modular_skyrat/modules/airlock_override/code/airlock_override.dm]: oh yeah, I have a signal for that

File: modular_skyrat/modules/cme/code/cme.dm [hunk 4]
```diff
@@ -133,15 +133,22 @@
...
 				priority_announce("Critical Coronal mass ejection detected! Expected intensity: [uppertext(cme_intensity)]. Impact in: [round((start_when * SSevents.wait) * 0.1, 0.1)] seconds. \
 				All synthetic and non-organic lifeforms should seek shelter immediately! \
 				Neutralize magnetic field bubbles at all costs.", "Solar Event", sound('modular_skyrat/modules/cme/sound/cme_warning.ogg'))
 			if(CME_ARMAGEDDON)
-				SSsecurity_level.set_level(SEC_LEVEL_GAMMA)
+				addtimer(CALLBACK(src, PROC_REF(cme_security_minimum_level), SEC_LEVEL_GAMMA), (round((start_when * SSevents.wait) * 0.1, 0.1)) SECONDS)
 				priority_announce("Neutron Mass Ejection Detected! Expected intensity: [uppertext(cme_intensity)]. Impact in: [round((start_when * SSevents.wait) * 0.1, 0.1)] seconds. \
 				All personnel should proceed to their nearest warpgate for evacuation, the Solar Federation has issued this mandatory alert.", "Solar Event", sound('modular_skyrat/modules/cme/sound/cme_warning.ogg'))
 
+/datum/round_event/cme/proc/cme_security_minimum_level(minimum_level)
```
Comment (Zonespace27) [inline modular_skyrat/modules/cme/code/cme.dm]: Codedoc

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/cme/code/cme.dm (hunk 4 above)
Comment (lessthnthree) [reply modular_skyrat/modules/cme/code/cme.dm]: Ah yes.

**Review by Gandalf2k15 — APPROVED** [review]
LGTM

**Pull request merged**